plugins/search-backend-module-elasticsearch/api-report.md

```diff
@@ -370,6 +370,7 @@ export type ElasticSearchSearchEngineIndexerOptions = {
   logger: Logger | LoggerService;
   elasticSearchClientWrapper: ElasticSearchClientWrapper;
   batchSize: number;
+  skipRefresh: boolean;
```

Comment:
As per @freben in the discord chat.

``` 
freben — Today at 12:18 PM
Hm could skipRefresh be optional? Makes the change non-breaking
And then checking options.skipRefresh !== true at the last lines
```

Reply:
makes sense.  hadn't checked there in a while.  Thanks.  Updated!